pkg/trace/agent/agent.go

```diff
@@ -287,7 +291,7 @@ func (a *Agent) Process(p *api.Payload) {
 
 var _ api.StatsProcessor = (*Agent)(nil)
 
-func (a *Agent) convertStats(in pb.ClientStatsPayload, lang, tracerVersion string) pb.StatsPayload {
+func (a *Agent) convertStats(in pb.ClientStatsPayload, lang, tracerVersion string) pb.ClientStatsPayload {
```

Comment:
```suggestion
func (a *Agent) processStats(in pb.ClientStatsPayload, lang, tracerVersion string) pb.ClientStatsPayload {
```

Reply:
You can still apply this change, no need to change anything else anywhere. We're not really converting anymore, just "processing"